### DISCUSSION TITLE
When will you support Polygon?

### DISCUSSION THREAD

**Discussion by 621625** [gochain/web3#212] [Q&A] [answered]
When will you support Polygon?

**Comment by treeder** [comment, marked as answer]
Probably already does, set `WEB3_RPC_URL=https://rpc-mainnet.matic.network` environment variable. 

eg:

```sh
export WEB3_RPC_URL=https://rpc-mainnet.matic.network
```

Then it should work.

**Reply by treeder** [reply]
@bitcoinmeetups did that work?

**Reply by scutdk** [reply]
It's working for sure. I've tried with both bsc and polygon.
